src/pages/index.js

```diff
@@ -30,49 +33,137 @@ const profileDescriptionInput = document.querySelector(
 );
 
 /*=============================================
-=           Create Cards          =
+=            server request            =
 =============================================*/
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "3235a097-fe3d-431f-b680-2a5fa7b7897e",
+    "Content-Type": "application/json",
+  },
+});
+
+let cardList;
+let userInfo;
+
+api
+  .loadPageResults()
+  .then(([card, userData]) => {
+    cardList = new Section(
+      {
+        items: [...card],
+        renderer: createCard,
+      },
+      ".cards__list"
+    );
+    cardList.renderItems();
+    userInfo = new UserInfo(
+      ".profile__title",
+      ".profile__description",
+      ".profile__image"
+    );
+    userInfo.setUserInfo({ title: userData.name, description: userData.about });
+    userInfo.setUserImage(userData.avatar);
+  })
+  .catch(console.error);
+
+/*=============================================
+=           Create Cards          =
+=============================================*/
+//fetchnewcard
 function createCard(cardData) {
   const cardElement = new Card(
     cardData,
     "#card-template",
-    handlePreviewPicture
+    handlePreviewPicture,
+    handleDeleteClick,
+    handleLikeClick
   );
   return cardElement.getView();
 }
 
 /*=============================================
-=            User Info            =
+=           profile Info            =
 =============================================*/
 
-const userInfo = new UserInfo(".profile__title", ".profile__description");
-
-function handleProfileEditSubmit(UserData) {
-  profilePopupForm.close();
-  userInfo.setUserInfo(UserData);
+function handleProfileEditSubmit(userData) {
+  profilePopupForm.setLoading(true);
+  api
+    .fetchEditProfile(userData)
+    .then(() => {
+      userInfo.setUserInfo({
+        title: userData.title,
+        description: userData.description,
+      });
+      profilePopupForm.close();
+    })
+    .catch((err) => console.error(err))
+    .finally(() => profilePopupForm.setLoading(false));
 }
 
 /*=============================================
-=            section            =
+=            profile image edit            =
 =============================================*/
 
-const cardList = new Section(
-  {
-    items: initialCards,
-    renderer: createCard,
-  },
-  ".cards__list"
+const profileImagePopupForm = new PopupWithForm(
+  "#picture-modal",
+  handleProfilePictureEdit
 );
-cardList.renderItems();
+profileImagePopupForm.setEventListeners();
 
-function handleAddCardEditSubmit(cardData) {
-  const name = cardData.title;
-  const link = cardData.url;
-  cardList.addItem({ name, link });
+const profileImageEditButton = document.querySelector(
+  ".profile-image__edit-button"
+);
+
+const form = document.querySelector("#change-profile-form");
+profileImageEditButton.addEventListener("click", (evt) => {
+  evt.preventDefault();
+  form.reset;
```

Comment:
`  form.reset;
`

It doesn't work if you don't call it


But... you don't need it here.

You need to **reset** the form only  in block `then` in order not to do that immediately after `submit` because when there is an error, the modal will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more.